Library/Homebrew/language/node.rb

```diff
@@ -4,6 +4,17 @@ def self.npm_cache_config
       "cache=#{HOMEBREW_CACHE}/npm_cache\n"
     end
 
+    def self.pack_for_installation
+      # npm 5.0.0 broke the assumption Homebrew made that npm would always
+      # create and install real directories when passed a file path. We can
+      # restore the old behaviour by packing the path first with "npm pack"
+      # which also helpfully spits out the filename it just created allowing
+      # us to pass that as the final argument.
```

Comment:
I was thinking about:
```ruby
      # Homebrew assumes the buildpath/testpath will always be disposable
      # and from npm 5.0.0 the logic changed so that when a directory is
      # fed to `npm install` only symlinks are created linking back to that
      # directory, consequently breaking that assumption. We require a tarball
      # because npm install creates a "real" installation when fed a tarball.
```
But not sure if that's still too wordy/full/etc.